src/views/v2/month/calendar-event/title.php

```diff
@@ -12,14 +12,17 @@
  * @version TBD
  *
  */
+$event    = $this->get( 'event' );
+$event_id = $event->ID;
 ?>
-<h3 class="tribe-events-calendar-month__calendar-event-title tribe-common-h8">
+<h3 class="tribe-events-calendar-month__calendar-event-title tribe-common-h8 tribe-events-tooltip"
```

Comment:
the tooltip stuff should be on the anchor rather than the heading, since that's what the user is interacting with to reveal the tooltip

Reply:
let's also use `data-js` attributes rather than the `.tribe-events-tooltip` class